s/elements/lookpad/style.css.ts

```diff
@@ -0,0 +1,13 @@
+
+import {css} from "@chasemoskal/magical"
+export const styles = css`
+
+:host {
+	display: block;
+	border: 1px solid;
+	touch-action: none;
+
+	max-width: var(--lookpad-max-width, 25rem);
+	aspect-ratio: var(--lookpad-aspect-ratio, 2/1);
```

Comment:
instead of this max-width and aspect ratio,

i think it might be simpler if the default lookpad was just a simple size that could be overwritten, like

```css
width: 10em;
height: 10em;
```